src/DB/projects.json

```diff
@@ -3524,6 +3524,25 @@
         "tech": ["HTML", "CSS", "JavaScript"]
       }
      
+    ]
+  },
+   {
+    "github_username": "prachisingh1008",
+    "Social_media": {
+      "gitHub": "https://github.com/prachisingh1008",
+      "LinkedIn": "https://www.linkedin.com/in/prachi-s-12845a212/",
+      "Instagram": "#",
+      "YouTube": "#",
+      "Twitter": "https://twitter.com/prachisingh1008"
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/prachisingh1008/Motivational-code-extension.git",
+        "title": "Motivational quote",
+        "description": "It displays page with a visually engaging background image of nature and displaying motivational quotes. This helps to create a more pleasant and inspiring environment for users when they open a new tab.",
+        "tech": ["HTML", "CSS", "JavaScript"]
+      }
```

Comment:
You don't need to add complete code for project addition. just add the code to the previous one. check the [contributing.md](https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md#3-add-following-code-to-end-of-srcdbprojectsjson) for more information

Reply:
May I know where I have added complete code , I have just added the repository link in the json file as mentioned earlier